src/phoenix/trace/span_evaluations.py

```diff
@@ -152,6 +157,24 @@ def __init_subclass__(
             tuple(sorted(prod)) for prod in product(*cls.index_names.keys())
         )
 
+    def to_parquet(self, path: Union[str, Path]) -> None:
+        table = Table.from_pandas(self.dataframe)
+        table = table.replace_schema_metadata(
+            {
+                **table.schema.metadata,
+                b"openinference": json.dumps({"eval_name": self.eval_name}).encode("utf-8"),
+            }
+        )
+        parquet.write_table(table, path)
```

Comment:
what's the name of the file? It needs to be addressable somehow. Is this meant to be in the path?

Reply:
Yes, I intended the caller to control the file name. Sounds like you have in mind saving to a hard-coded file path?

Reply:
The alternative I think is to have the file name be a UUID - say of the object itself. Keeping it under our control could have some benefits I think.